rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2084,6 +2084,25 @@ Map<String, String> getStatisticsOnMyAccount(@HeaderParam("sessionid")
     List<JobUsageData> getUsageOnMyAccount(@HeaderParam("sessionid") String sessionId,
             @QueryParam("startdate") Date startDate, @QueryParam("enddate") Date endDate) throws RestException;
 
+    /**
+     * Returns an object containing jobs number and statistics based on the given filters
```

Comment:
Returns an object containing jobs count-by-state and statistics based on the given filters